src/session/AppSessionStateContext.tsx

```diff
@@ -35,6 +35,7 @@ export function AppSessionStateProvider({
         isTheSameSession(previousState, newState) ? previousState : newState
       );
     };
+    appSessionStateClient.onSessionUpdate();
```

Comment:
👍 

We should apply the same change in [Editors WebApp](https://github.com/FromDoppler/doppler-editors-webapp/blob/main/src/components/AppSessionStateContext.tsx): [DE-704](https://makingsense.atlassian.net/browse/DE-704)